lib/config/options/index.ts

```diff
@@ -2359,6 +2359,19 @@ const options: RenovateOptions[] = [
     default: 24 * 60,
     globalOnly: true,
   },
+  {
+    name: 'cacheTtlOverride',
+    description: 'An object that contains cache namespace TTL override values',
+    type: 'object',
+    stage: 'repository',
+    default: {},
+    additionalProperties: {
+      type: 'number',
+    },
```

Comment:
not sure I'm using this field as intended. 
my assumption is that it corresponds to the Record key type.

Reply:
Check the generated JSON schema to see if it validates how you expect

Reply:
it doesnt provide any additional type safety, and the schema looks the same.
so i've removed it.